pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -590,19 +590,19 @@ in
 
   cord-nvim =
     let
-      version = "2024-07-19";
+      version = "unstable-2024-09-26";
```

Comment:
```suggestion
      version = "0-unstable-2024-09-26";
```
This is the new convention for the unstable versions.

Reply:
Ugh, I will have to adapt #336137 for this...

Reply:
> This is the new convention for the unstable versions.

Thanks! I updated it to match the new convention